File: backend/models/presidentModel.js
const mongoose = require('mongoose');
const ModelFactory = require('./modelFactory');

const PresidentSchema = new mongoose.Schema({
  president_id: String,
  first_name: String,
  last_name: String,
  age: Number,
  president_info: String,
  president_image: String,
});

const President = ModelFactory.createModel('presidents', PresidentSchema);

module.exports = President;

File: InfoWarOSMobilUygulamasi/InfoWar/InfoWar/Model/PresidentModel.swift
//
//  PresidentModel.swift
//  InfoWar
//
//  Created by Serhat  Şimşek  on 4.01.2024.
//

import Foundation

struct PresidentModel: Decodable {
    let id, presidentId, firstName, lastName: String?
    let age: Int?
    let presidentInfo: String?
    let presidentImage: String?
}
